presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcPageSink.java

```diff
@@ -165,6 +165,13 @@ else if (javaType == Slice.class) {
             throw new PrestoException(JDBC_NON_TRANSIENT_ERROR, e);
         }
         catch (SQLException e) {
```

Comment:
I'd add a comment in code noting what this is for, like "convert chained SQLExceptions to suppressed exceptions so they are visible in the stack trace."